src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const sourceObject = {};
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    const [property, value] = declaration.split(':');
```

Comment:
When splitting 'declaration' by ':', there might be cases where property values contain colons, such as in URLs or content properties. This code will not handle those cases correctly. A more robust solution would be to split at the first colon only.